interface/themes/oemr-bs3-override.css

```diff
@@ -0,0 +1,79 @@
+/*override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
```

Comment:
The reason this file was created was to be a placeholder for styling that was needed to style the 28 odd pages and minimize use of internal stylesheets.  It is  a start and not the end. I agree that some of this can be consolidated into core.css. 
As always I have tried to keep my changes separate in order to easily identify them and modify as needed.  Some of the styling that was included in here is to ensure that all stylesheets currently in use will display the pages in a consistent way.  Once more pages get styled we will be able to identify what would constitute a 'core' style.
In the development phase the oemr-bs3-override.css is basically a copy and paste operation using  @include and may eventually disappear by being absorbed into core.css or individual stylesheets.
It would certainly help if we include name and ID for elements to help in targeting them for styling without having to modify the code.
A wiki page with the basic styling pertinent to openEMR can be written that would contain styling tips to help convert current non-bootstrap pages and design future pages.